static/main.html

```diff
@@ -61,16 +46,16 @@ <h4>States locked</h4>
         </thead>
         <tbody>
             <tr ng-repeat="r in results.states">
-                <td><span ng-if="isLocked(r.path)" ng-init="lock=locks[r.path]" class="glyphicon glyphicon-lock" title="Locked by {{lock.Who}} on {{lock.Created | date:'medium'}} ({{lock.Operation}})"></span></td>
-                <td><span class="glyphicon glyphicon-link" aria-hidden="true"></span> <a href="state/{{r.path}}?versionid={{r.version_id}}">{{r.path}}</a></td>
+                <td><span class="glyphicon glyphicon-link" aria-hidden="true"></span> <a href="lineage/{{r.lineage_value}}?versionid={{r.version_id}}">{{r.path}}</a></td>
```

Comment:
Why does this remove the lock icon from this view?

Reply:
I moved it previously to `lineage.html` but since this view was ultimately not suitable, I put everything back in this file and I must have forgotten this element. I fix this.